app.js

```diff
@@ -1,15 +1,22 @@
 let myTodo = [];
 let taskTitle = "";
+let isInEditMode = false;
 /*----------DOM element reference----------*/
 let todoInputBox = document.getElementById("todo-input");
 let addButton = document.getElementById("add-todo");
 let activeTodoList = document.getElementById("todo-item");
 let completedTodoList = document.getElementById("finalTodo");
+let addEditedTodoBox = document.getElementById("edit-todo");
 /*----------DOM element reference----------*/
+let selectedIndex = -1;
 
 const onChangeHandler = (event) => {
   if (event.key === "Enter") {
-    createToDo();
+    if (selectedIndex == -1) {
```

Comment:
@RiyaBasak26 can you please explain this codeblock?

Reply:
this selectedIndex helping us to check on enter event it will create Todo or replace the edited todo